### ISSUE TITLE
Orgs - Edit your organization membership: "state" is an enum

### ISSUE THREAD

**Issue by gr2m** [octokit/routes#98]
- endpoint scope: Organizations
- endpoint name: Edit your organization membership
- route: `PATCH /user/memberships/orgs/:org`
- documentation URL: https://developer.github.com/v3/orgs/members/#edit-your-organization-membership

### Current specification

https://github.com/octokit/routes/blob/a509e1a89672025ec580405648b9c5666e3e7b06/routes/orgs/members/edit-your-organization-membership.json#L13-L18

### I expected

<details>
  <summary>params is missing the "foo" option</summary>

```json
{
  "name": "Edit your organization membership",
  "enabledForApps": false,
  "method": "PATCH",
  "path": "/user/memberships/orgs/:org",
  "params": [
    {
      "name": "org",
      "type": "string",
      "required": true,
      "description": ""
    },
    {
      "name": "state",
      "type": "string",
      "description": "The state that the membership should be in. Only `\"active\"` will be accepted.",
      "required": true,
      "enum": [
        "active"
      ]
    }
  ],
  "description": "",
  "documentationUrl": "https://developer.github.com/v3/orgs/members/#edit-your-organization-membership"
}
```
</details>

---

If you’d like to work on it, just comment below and follow instructions in [CONTRIBUTING.md](https://github.com/octokit/routes/blob/master/CONTRIBUTING.md)

**Comment by knrt10** [comment]
I would like to work on this.

**Comment by gr2m** [comment]
fixed via #105
